File: II - Linguaggi di programmazione e laboratorio/Python/Exercises/29a_compito_java_20190411/Dipendente.py
class Dipendente:
    def __init__(self, codice, tipo, nome, costo):
        self._codice = codice
        self._tipo = tipo
        self._nome = nome
        self._costo = costo
        self._numero_servizi = 0

    def get_costo(self):
        return self._costo

    def incrementa_numero_servizi(self):
        self._numero_servizi += 1

    def get_numero_servizi(self):
        return self._numero_servizi

    def get_nome(self):
        return self._nome

    def __str__(self):
        return self._nome + "\t" + str(self._codice)

File: II - Linguaggi di programmazione e laboratorio/Java/Exercises/29_compito_java_2019_04_11a/Dipendente.java
public class Dipendente {
	protected int codice;
	protected String tipo;
	protected String nomeDipendente;
	protected double costo;
	
	public Dipendente(int codice, String tipo, String nomeDipendente, double costo) {
		this.codice = codice;
		this.tipo = tipo;
		this.nomeDipendente = nomeDipendente;
		this.costo = costo;
	}
	
	public double getCosto()
	{
		return costo;
	}
		public int getCod()
	{
		return codice;
	}
	public String getNome()
	{
		return nomeDipendente;
	}
}